modules/cli/src/main/java/org/apache/ignite/internal/cli/logger/CliLoggers.java

```diff
@@ -59,6 +61,10 @@ public static IgniteLogger forName(String name) {
         return Loggers.forName(name, loggerFactory);
     }
 
+    public static void addApiClient(ApiClient client) {
+        httpLoggers.add(new HttpLogging(client));
```

Comment:
`HttpLogging` does not define `equals` and `hashCode`. So, I think the method `addApiClient ` is not idempotent. I suppose we will add client loggers each time we create a client. Could you confirm that using `--verbose` flag does not duplicate the HTTP logging in this case?

Reply:
I guess you're right. I can use similar map from path to logger here as in the client factory.

Reply:
Or just define `hashcode` and `equals` that use URL as a reference.